tdvt/tdvt/config_gen/test_config.py

```diff
@@ -250,6 +250,35 @@ def build_config_name(prefix, dsname):
 def build_tds_name(prefix, dsname):
     return prefix + dsname + '.tds'
 
+class TabQueryPath(object):
+    sep = ';'
+
+    def __init__(self, linux_path, mac_path, windows_path):
+        self.linux_path = linux_path
```

Comment:
I've been wondering if it would be worth it to have TDVT start to use `Path` objects throughout. That would reduce some code complexity since `Path` has [useful methods](https://docs.python.org/3/library/pathlib.html#methods) like `.exists()` and `is_file()`. We've started doing that in the connector packager library, and it's been pretty useful I think.

Reply:
I think its a good idea. I had a bit of problem trying to store two posix paths and a windows path. I'll look into the idea of using them though.